posts/tests.py

```diff
@@ -1,3 +1,61 @@
-# from django.test import TestCase
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.authtoken.models import Token
+from rest_framework.test import APITestCase
 
-# Create your tests here.
+from posts.models import Post
+from users.models import Followers, User
+
+
+class PostTest(APITestCase):
+    def setUp(self):
+        self.signup_data = {"username": "harrypotter", "password": "hogwarts934"}
+        self.signup_data_another = {
+            "username": "robinsoncrusoe",
+            "password": "daniel1719",
+        }
+        self.client.post(reverse("signup"), self.signup_data)
+        self.client.post(reverse("signup"), self.signup_data_another)
+        post = Post(
+            photo="data:image/gif;base64,R0lGODlhEAAOALMAAOazToeHh0tLS/7LZv/0jvb29t"
+            "/f3//Ub//ge8WSLf/rhf/",
+            description="picture",
+            user=User.objects.get(username=self.signup_data.get("username")),
+        )
```

Comment:
Why do you create users with the API call but create a post directly via the database?

Reply:
Also, let's better use IDs to get entities

Reply:
Done, now posts also can be created with the API